src/app/dashboard/views/DiscoveriesList/DiscoveriesList.tsx

```diff
@@ -27,8 +27,12 @@ import type { Discovery } from "app/store/discovery/types";
 import { DiscoveryMeta } from "app/store/discovery/types";
 import type { RootState } from "app/store/root/types";
 
-export enum Label {
+export enum Labels {
   Title = "Discoveries list",
```

Comment:
Would be good to update this too so that we at least don't have inconsistencies within the same file.
```suggestion
  DiscoveriesList = "Discoveries list",
```

Reply:
Done!